src/bicep/examples/newWorkload/newWorkload.bicep

```diff
@@ -1,30 +1,55 @@
 targetScope = 'subscription'
+/*
 
-param mlzDeploymentVariables object = json(loadTextContent('../deploymentVariables.json'))
+  PARAMETERS
+
+  Here are all the parameters a user can override.
+
+  These are the required parameters that Mission LZ Tier 3 workload does not provide a default for:
+    - resourcePrefix
+
+*/
+
+// REQUIRED PARAMETERS
+
+@minLength(3)
+@maxLength(10)
+@description('A prefix, 3 to 10 characters in length, to append to resource names (e.g. "dev", "test", "prod", "mlz"). It defaults to "mlz".')
+param resourcePrefix string
```

Comment:
I updated this to match the convention used in `mlz.bicep`.  Let me know if these are changes are correct by resolving this conversation.

https://github.com/Azure/missionlz/blob/199b986c770d0d331b6d99fce39d9b9807af2e28/src/bicep/mlz.bicep#L16-L19

Reply:
perfect.